samples/parcel/package.json

```diff
@@ -8,9 +8,6 @@
         "build": "parcel build index.html",
         "playwright:webserver": "parcel index.html --port 3000"
     },
-    "alias": {
-        "./MonacoEnvironment.ts": "MonacoEnvironment"
-    },
```

Comment:
Vestigial

Left over from a different attempt to get the Parcel example working.